app/models/smart_proxy.rb

```diff
@@ -97,6 +97,7 @@ def statuses
       end
     end
     @statuses[:version] = ProxyStatus::Version.new(self)
+    @statuses[:logs] = ProxyStatus::Logs.new(self)
```

Comment:
Oh sorry, no. This needs to be here because the generic code use `humanize` which does not work in this case (it's `Log` vs `Logs`).